packages/react/src/index.js

```diff
@@ -10,49 +10,50 @@ const createCss = (init) => {
 	const sheet = createCoreCss(init)
 
 	return assign(sheet, {
-		/** Returns a React component. */
+		/** Returns a React forwarded ref object. */
 		styled: (
 			/** Type of component. */
 			...inits
 		) => {
-			const defaultType = inits.map((init) => (Object(init).type ? init.type : init)).find((init) => init) || 'span'
+			const defaultType = inits.map((init) => (Object(init).stitchesType ? init.stitchesType : init)).find((init) => init) || 'span'
 			const composition = sheet.css(...inits.filter((init) => $$composers in Object(init) || (init && typeof init === 'object' && !init.$$typeof)))
```

Comment:
This insinuates that `styled` can take more than more than 2 arguments, like `styled('div', { color: 'green' }, { fontWeight: 'bold' })`. I thought this might be legacy as it's not part of the documented API, so I tried simplifying it to just the two arguments, but it failed a couple of tests and I didn't have time to investigate.